test/unit/factories/Axis.mocha.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path='../test.mocha.ts' />
+
+describe('n3Charts.Factory.Axis', () => {
+  var domElement: JQuery = angular.element(document.body).append('<div></div>');
+  var axis: n3Charts.Factory.Axis = undefined;
+  var eventMgr: n3Charts.Utils.EventManager = undefined;
+  var factoryMgr: n3Charts.Utils.FactoryManager = undefined;
```

Comment:
Yeah, the idea behind this was that when I launched the examples, I got an error because Axis was trying to access the `dim` property on Container. So I tried to fix this in TDD mode. But I agree with you, let's try to stay unitary in unit tests ! I'm leaving this one as it is for now, we'll remove the extra code next time we work on this class (that shouldn't be too long !)